### PR TITLE
[INLONG-2827][Manager] Support configurable plugin when creating Hive table

### LINKED ISSUE #2827: [Feature] Manager support configurable plugin when creating hive table.
### Description

Manager support configurable plugin when creating hive table.

The operation event of creating hive table include private authentication check and the third-party hive.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by luchunliang**
### Title Name: [INLONG-2827][Manager] Support configurable plugin when creating Hive table

Fixes #2827 

### Motivation

Support configurable plugin when creating Hive table.

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [X] Make sure that the change passes the CI checks.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/utils/SpringContextUtils.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
+
+/**
+ * 
+ * SpringContextUtils
+ */
+/**
```
[the remark this answers is not in the document]
Comment (luchunliang) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/utils/SpringContextUtils.java]: remove it.

**Review by vernedeng — APPROVED** [review]

**Review by wardlican — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/utils/SpringContextUtils.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            LOG.error(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * getBean
+     * 
+     * @param  beanName
+     * @param  defaultClass
+     * @return              Object
```
Comment (wardlican) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/utils/SpringContextUtils.java:71]: useless space

Comment (luchunliang) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/utils/SpringContextUtils.java:71]: fixed it

**Comment by healchow** [comment]
Please refer to this Discussions (https://github.com/apache/incubator-inlong/discussions/3082) to modify the code style.

**Review by healchow — APPROVED** [review]

**Pull request merged**